scripts/js/lib/api/updateLinks.test.ts

```diff
@@ -200,6 +200,7 @@ test("normalizeUrl()", () => {
     `#qiskit_ibm_runtime.RuntimeJob.job`,
     `qiskit_ibm_runtime.RuntimeJob#qiskit_ibm_runtime.RuntimeJob.run`,
     `stubs/qiskit_ibm_runtime.RuntimeJob`,
+    `#_CPPv415qk_obs_identity8uint32_t`,
```

Comment:
Hm, this is going to fail with https://github.com/Qiskit/documentation/pull/2793. I think that suggests that `updateLinks.ts` needs to handle it? Your PR is already pretty big, so maybe create an issue in open source with this bug and ask Frank to take a look?

Reply:
Ah, I see. I was confusing the test with what is happening in production. Sg to keep this. Thanks for clarifying / pardon my misread!